### PR TITLE
GSdx: Switch tooltips to unicode

### PR DESCRIPTION
**Pull request by AmFobes**
I didn't test the tooltips on #4188 (oops)

### CODE REVIEW

File: plugins/GSdx/Window/GSSetting.cpp [hunk 1]
```diff
@@ -25,30 +25,36 @@
 #include "resource.h"
 #endif
 
-const char* dialog_message(int ID, bool* updateText) {
+#ifdef _WIN32
+#define _cvtString( s ) L##s
```
Comment (turtleli) [inline plugins/GSdx/Window/GSSetting.cpp:29]: Leading underscores shouldn't be used. They're reserved for use by the standard for implementation use.

**Review by GovanifY — APPROVED** [review]

File: plugins/GSdx/Window/GSSetting.cpp [hunk 2]
```diff
@@ -25,30 +25,36 @@
 #include "resource.h"
 #endif
 
-const char* dialog_message(int ID, bool* updateText) {
+#ifdef _WIN32
+#define cvtString( s ) L##s
+const wchar_t* dialog_message(int ID, bool* updateText) {
+#else
+#define cvtString( s ) s
+const char* dialog_message(int ID, bool* updateText){
```
Comment (lightningterror) [inline plugins/GSdx/Window/GSSetting.cpp:33]: Formatting can be better on these lines, above ones too.

File: plugins/GSdx/Window/GSSetting.cpp [hunk 3]
```diff
@@ -215,6 +221,7 @@ const char* dialog_message(int ID, bool* updateText) {
 		default:
 			if (updateText)
 				*updateText = false;
-			return "";
+			return cvtString("");
 	}
 }
+#undef cvtString
```
Comment (turtleli) [inline plugins/GSdx/Window/GSSetting.cpp:227]: Is this line necessary?

Comment (AmFobes) [reply plugins/GSdx/Window/GSSetting.cpp:227]: Not necessarily although intellisense didn't complain when I tried to use the define elsewhere which led me to believe that it'll clutter up stuff.

Comment (turtleli) [reply plugins/GSdx/Window/GSSetting.cpp:227]: Ok. I haven't coded for a while and don't recall what's best. I guess it might be better to leave it there.

**Review by RedDevilus — APPROVED** [review]

File: plugins/GSdx/Window/GSSetting.cpp [hunk 4]
```diff
@@ -25,30 +25,36 @@
 #include "resource.h"
 #endif
 
-const char* dialog_message(int ID, bool* updateText) {
+#ifdef _WIN32
+#define cvtString(s) L##s
+const wchar_t* dialog_message(int ID, bool* updateText) {
+#else
+#define cvtString(s) s
+const char* dialog_message(int ID, bool* updateText){
```
Comment (lightningterror) [inline plugins/GSdx/Window/GSSetting.cpp:33]: You can also move the brackets to new line for consistency.

**Review by lightningterror — APPROVED** [review]

**Pull request merged**